### PR TITLE
Add IE11 to Document.documentElement compatibility table

### PR DESCRIPTION
**Pull request by p2kmgcl**
Pasted_image_at_2019-04-29__4_42_PM

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
Hey there, welcome to BCD!  I was just checking up on your PR, however I've been able to find support back in IE 9, and possibly even older versions.  I'll be testing for an official version number.  :wink:

File: api/Document.json [hunk 1]
```diff
@@ -2458,7 +2458,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "11"
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
              "version_added": "5"
```

**Comment by p2kmgcl** [comment]
@ddbeck @vinyldarkscratch sorry for the delay, suggestion applied. Do I need to squash everything?

**Comment by vinyldarkscratch** [comment]
Thank you for the updates!  I'm happy to say that we're one of those repositories that performs squash merging.  It makes it much easier to handle pull requests, and keeps the commits on master clean without sacrificing the history whilst the pull request is open.  :wink:

Everything looks good, and once again, welcome to BCD!

**Pull request merged**